src/main/java/nl/hsac/fitnesse/fixture/slim/SlimFixture.java

```diff
@@ -168,8 +168,9 @@ protected boolean repeatUntil(RepeatCompletion repeat) {
                 waitMilliseconds(nextInterval);
 
                 loopTimer.start();
-                repeat.repeat();
+
                 try {
+                    repeat.repeat();
```

Comment:
Doesn't moving this line break backwards compatibility in situations where the 'repeat' throws an exception for another reason (e.g. unable to connect to host)